tests/unit/components/AppSidebar/AppSidebarTabs.spec.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ *
+ * @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { mount } from '@vue/test-utils'
+
+import Vue from 'vue'
+import AppSidebarTabs from '../../../../src/components/AppSidebarTabs/AppSidebarTabs.vue'
+import AppSidebarTab from '../../../../src/components/AppSidebarTab/AppSidebarTab.vue'
+import ActionButton from '../../../../src/components/ActionButton/ActionButton.vue'
+
+describe('AppSidebarTabs.vue', () => {
```

Comment:
Oh wow, with tests?? :heart_eyes:

Reply:
Yeah, some tests I wrote, but mainly to check for #733. There could be much more tests 😉

Reply:
Yes, unit tests are not my strength to be honest, I should really take the time to dive into this